docs/SDL/OnAppPermissionChanged/index.md

```diff
@@ -17,12 +19,17 @@ Purpose
 |:---|:---|:--------|:---------|
 |appID|Integer|true||
 |isAppPermissionsRevoked|Boolean|false||
-|appRevokedPermissions|[Common.PermissionItem](../../common/structs/#permissionitem)|false|array: true<br>minsize: 1<br>maxsize: 100|
+|appRevokedPermissions|[Common.PermissionItem]|false|array: true<br>minsize: 1<br>maxsize: 100|
 |appRevoked|Boolean|false||
 |appPermissionsConsentNeeded|Boolean|false||
 |appUnauthorized|Boolean|false||
-|priority|[Common.AppPriority](../../common/enums/#apppriority)|false||
-|requestType|[Common.RequestType](../../common/enums/#requesttype)|false|array: true<br>minsize: 0<br>maxsize: 100|
+|priority|[Common.AppPriority]|false||
+|requestType|[Common.RequestType]|false|array: true<br>minsize: 0<br>maxsize: 100|
+
+
+[Common.PermissionItem]: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Structs/index.md#permissionitem
+[Common.AppPriority]: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Enums/index.md#apppriority
+[Common.RequestType]: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Enums/index.md#requesttype
```

Comment:
Should be relative links (look at where the original links pointed for reference)